web/src/stream_settings_ui.js

```diff
@@ -658,7 +667,7 @@ export function setup_page(callback) {
         // is only useful if the user has permission to create
         // streams, either explicitly via user_can_create_streams, or
         // implicitly because realm.realm_is_zephyr_mirror_realm.
-        $("#stream_filter input[type='text']").on("keypress", (e) => {
+        $("#search_stream_name").on("keypress", (e) => {
```

Comment:
Why is this change part of this commit?

Reply:
Oh my bad. Will incorporate separate logical commit for this one.